test/fetch-server.js

```diff
@@ -1,5 +1,18 @@
-describe(`FetchServer`, function() {
-  it(`runs this test`, function() {
-    expect(true).to.be.ok;
+import {expect} from '@esm-bundle/chai';
+import {FetchServer} from '../lib';
+
+describe(`FetchServer`, () => {
+  it(`runs this test`, async () => {
```

Comment:
Thanks for making a real test case!
```suggestion
  it(`stubs an API endpoint response with a static return value`, async function() {
```